reflex/utils/prerequisites.py

```diff
@@ -279,6 +282,18 @@ def get_app(reload: bool = False) -> ModuleType:
         raise
 
 
+def get_app(reload: bool = False) -> App:
```

Comment:
can we make this new function `get_app_instance` and leave `get_app` the way it was, to retain as much compatibility as possible